binding.c

```diff
@@ -330,7 +331,12 @@ NAPI_METHOD(udx_napi_send) {
 
   uv_buf_t b = uv_buf_init(buf, buf_len);
 
-  err = udx_send(req, self, &b, 1, (const struct sockaddr *) &addr, on_udx_send);
+  if (ttl == 0) {
+    err = udx_send(req, self, &b, 1, (const struct sockaddr *) &addr, on_udx_send);
+  } else {
+    err = udx_send_ttl(req, self, &b, 1, (const struct sockaddr *) &addr, ttl, on_udx_send);
+  }
+
```

Comment:
this if isn't needed now is it? just call _ttl always?

Reply:
For reference:

https://github.com/mafintosh/libudx/blob/b90232ba01557bee396274294b641f45fca43e7d/src/udx.c#L774

We could specifically allow `ttl == 0`.

Reply:
Yea I think we should allow 0 also